backend-worker/worker.py

```diff
@@ -427,26 +432,32 @@ def pixelpercents_under_parcel(parcel_wkt_epsg3857, source_raster_path):
     return_values = {}
     n_values_under_parcel = numpy.sum(counts)
     for lulc_code, pixel_count in zip(values_under_parcel, counts):
-        return_values[lulc_code] = round(
+        # cast lulc_codes to int from numpy_int16 for future json dump call
+        # which does not allow numpy types for keys
+        return_values[int(lulc_code)] = round(
             pixel_count / n_values_under_parcel, 4)
 
     return return_values
 
 
 def do_work(ip, port):
     local_appdata_dir = 'appdata'
-    job_queue_url = f'{ip}:{port}/jobsqueue/'
+    job_queue_url = f'http://{ip}:{port}/jobsqueue/'
     LOGGER.info(f'Starting worker, queueing {job_queue_url}')
     LOGGER.info(f'Polling the queue every {POLLING_INTERVAL_S}s if no work')
     while True:
         response = requests.get(job_queue_url)
-        if not response.json:
+        # if there is no work on the queue, expecting response.json()==None
+        if not response.json():
```

Comment:
Huh, I thought `.json` was an attribute rather than a function ... thanks for catching!